code/game/machinery/bank_machine.dm

```diff
@@ -45,19 +46,19 @@
 		return
 	if (machine_stat & (BROKEN|NOPOWER))
 		say("Insufficient power. Halting siphon.")
-		end_syphon()
+		end_siphon()
 		return
 	var/siphon_am = 100 * delta_time
 	var/datum/bank_account/D = SSeconomy.get_dep_account(ACCOUNT_CAR)
 	if(!D.has_money(siphon_am))
 		say("Cargo budget depleted. Halting siphon.")
-		end_syphon()
+		end_siphon()
 		return
 
 	playsound(src, 'sound/items/poster_being_created.ogg', 100, TRUE)
 	syphoning_credits += siphon_am
 	D.adjust_money(-siphon_am)
-	if(next_warning < world.time && prob(15))
+	if(unauthorized && next_warning < world.time && prob(15))
 		var/area/A = get_area(loc)
 		var/message = "Unauthorized credit withdrawal underway in [initial(A.name)]!!"
```

Comment:
I think it'd be better if you changed it so that the message was different for authorised personnel. Do remember that this is the station's budget as well as the cargo's budget. Captains purchasing shuttles, for example, draws from this budget.

Reply:
So like, still say something in common even when it's authorized?

Reply:
yes